src/CoreBundle/Resources/translations/messages.en.xliff

```diff
@@ -225,7 +225,6 @@
                 <source>chameleon_system_core.template_tool.show_dynamic_box</source>
                 <target>show</target>
             </trans-unit>
-
```

Comment:
Please revert this change (add the line again).

Reply:
ok